app/src/main/res/layout/row_loan_account.xml

```diff
@@ -1,14 +1,19 @@
 <?xml version="1.0" encoding="utf-8"?>
 
-<LinearLayout
-    xmlns:android="http://schemas.android.com/apk/res/android"
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
     android:minHeight="?attr/listPreferredItemHeight"
     android:orientation="horizontal"
     android:paddingLeft="@dimen/row_padding_left"
-    android:paddingRight="@dimen/row_padding_right"
->
+    android:paddingRight="@dimen/row_padding_right">
+
+    <View
+        android:id="@+id/vw_loanStatus"
+        android:layout_width="25dp"
+        android:layout_height="25dp"
+        android:layout_gravity="center"
+        android:background="@drawable/closed" />
```

Comment:
This is good but I found, as we will add account type. we have to add more shape drawable.
that is not good.

Can you, please change this View to circuler ImageView using this https://github.com/openMF/android-client/blob/master/mifosng-android/src/main/java/com/mifos/mifosxdroid/views/CircularImageView.java
custom Circular ImageView.

and Add  CircularImageView in xml.